.github/workflows/bazel_example.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     - name: Build example package; all targets
       run: |
         cd bazel_example;
-        bazel build ...
+        bazel build //...
```

Comment:
Without this change also it works fine. I referred this change in my initial PR and there its working fine

Reply:
The later is more powerful though in this context both would have same meaning

Reply:
I meant `//...` searches for build targets in the entire workspace not just subdirectories.

Reply:
reverted